src/Symfony/Component/Translation/Translator.php

```diff
@@ -392,6 +397,10 @@ private function loadFallbackCatalogues($locale): void
 
     protected function computeFallbackLocales($locale)
     {
+        if (null === $this->parentLocales) {
+            $parentLocales = \json_decode(\file_get_contents(__DIR__.'/Resources/data/parents.json'), true);
```

Comment:
we don't fully-qualify all function calls in Symfony (to reduce the visual clutter). We only fully-qualify the function which can be optimized by the compiler (as the compiler cannot optimize them when it does not know for sure that the global function is the one being called)